idea-plugin/build.gradle

```diff
@@ -28,17 +28,14 @@ intellij {
 patchPluginXml {
```

Comment:
should we also bump the version of intellij we test with?

Reply:
We can do that later.
I know for a fact it doesn't work fully well in 193.4099+ and keeps throwing exceptions, but need to figure out why and how to fix. Funnily enough their automatic detection hasn't picked that up.